src/libpriv/rpmostree-core.c

```diff
@@ -2203,6 +2205,10 @@ rpmostree_context_assemble_commit (RpmOstreeContext      *self,
         if (parent_version)
           g_variant_builder_add (&metadata_builder, "{sv}", "version",
                                  g_variant_new_string (parent_version));
+
+        /* Flag the commit as a client layer */
+        g_variant_builder_add (&metadata_builder, "{sv}", "rpmostree.clientlayer",
+                               g_variant_new_boolean (TRUE));
```

Comment:
Could we just always include the enum value itself in e.g. `"rpmostree.assemble_type"` ?